src/target/common/devo/protocol/pwm.c

```diff
@@ -18,58 +18,48 @@
   #pragma long_calls
 #endif
 
-#include "common.h"
-#ifndef DISABLE_PWM
 
 #include <libopencm3/stm32/rcc.h>
-#include <libopencm3/stm32/gpio.h>
 #include <libopencm3/stm32/timer.h>
+#include <libopencm3/stm32/gpio.h>
+#include <libopencm3/stm32/dma.h>
```

Comment:
sorry includes please.

Reply:
typo in my comment

Reply:
Seems like a good idea, though I couldn't find much about best practices online.  Reordered in the spirit of [these guidelines.](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes)